packages/client-config/src/client-config-writer/client_config_to_mobile_legacy_converter.ts

```diff
@@ -80,7 +82,9 @@ export class ClientConfigMobileConverter {
           Scopes: clientConfig.oauth.scope,
           SignInRedirectURI: clientConfig.oauth.redirectSignIn,
           SignOutRedirectURI: clientConfig.oauth.redirectSignOut,
-          AppClientId: clientConfig.oauth.clientId,
+          AppClientId:
```

Comment:
can you provide some context here, oauth.clientId was not working, but now we are still accepting it in some scenarios?

Reply:
`clientConfig.oauth.clientId` might be empty because `clientConfig.oauth.clientId` is provided by user in when configuring `defineBackend`.  It's not required in the [schema](https://docs.amplify.aws/react/reference/amplify_outputs/).

Reply:
I think we can remove `clientConfig.oauth.clientId` since it's always undefined